exercises/forth/canonical-data.json

```diff
@@ -17,13 +17,6 @@
           "1 2 3 4 5"
         ],
         "expected": [1, 2, 3, 4, 5]
-      },
-      {
-        "description": "all non-word characters are separators",
-        "input": [
-          "1\u00002\u00133\n4\r5 6\t7"
```

Comment:
> replace the unicode representation of the characters with their hex equivalent

Is that possible? I don't think `\x` is allowed according to http://www.json.org/, though if you have information to the contrary, it would be good to have it. I suggest to take option 3 if possible, else take option 2.

> Also I had a look at the description.md and I'll need to update it too:

Ah, true. Amusing that unicode digits is mentioned, since I didn't see any non-ascii digits being used in any language's test suite...

> add a "integrating feedback" commit

I highly dislike those since the break the rule of "one logical change per commit" and the commit message tells me nothing about what changed.

Since the changes being made are small, I do not object to history being rewritten.

If there are many changes being made and then a reviewer has requested a small piece of feedback, then it is helpful to address the feedback in a separate commit simply so the reviewer can understand what has changed since the last review. But I would almost always squash those before merging, since it is likely those address-feedback commits do not stand on their own but only make sense in the context of the commit they amend.

These are my preferences; I don't know if the project as a whole has chosen some standards yet.